test/simpletest.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var expect = require('chai').expect;
+var greeting = require('../simple');
+
+describe('greeting function', function(name) {
+  it('should return hello name', function() {
```

Comment:
`name` is undefined here. the test should use a specific name, like you did above:
`expect(greeting.greet('Tricia')).to.eql('Hello Tricia');`